var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -580,6 +608,7 @@ def enable_or_disable(variant, keyword="USE", var=None):
         # https://github.com/pytorch/pytorch/issues/111526
         # https://github.com/pytorch/pytorch/issues/124018
         env.set("USE_FLASH_ATTENTION", "OFF")
```

Comment:
Note that I'm changing this in https://github.com/spack/spack/pull/48521. Does `USE_MEM_EFF_ATTENTION` have  similar issues with memory during compilation?

Reply:
Removed USE_MEM_EFF_ATTENTION flag change.